src/sidebar/components/ThreadList.tsx

```diff
@@ -120,7 +120,7 @@ export default function ThreadList({ threads }: ThreadListProps) {
 
     // We currently assume that the scroll container's size only changes with
     // the window as a whole. A more general approach would involve using
-    // ResizeObserver via the `observeElementSize` utility.
```

Comment:
The `observeElementSize` utility has been removed because only the `Excerpt` component was effectively using it, so I have removed its mention in this comment.